cloud/helm-chart/values.yaml

```diff
@@ -0,0 +1,50 @@
+replicaCount: 1
+
+image:
+  repository: ghcr.io/aklivity/zilla
+  pullPolicy: IfNotPresent
+  tag: latest
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+
+securityContext: {}
+
+livenessProbePort: 0
+readinessProbePort: 0
+
+args:
+  - start
+  - -v
+  - -e
```

Comment:
Do you think it makes sense to include the `-v` (verbose) for a production helm chart?

Reply:
I don't think it's bad to see what's going on in a prod system. Unless there is some concrete disadvantage (e.g. performance issue), I'd keep it.